generated/nidcpower/nidcpower_service.cpp

```diff
@@ -14,6 +14,8 @@
 
 namespace nidcpower_grpc {
```

Comment:
Run the MI system tests (that work on simulation) or let me know and I can run them

Reply:
(@astarche ran these and reported they passed)